src/actions/helpers.js

```diff
@@ -1,50 +1,58 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+// @flow
 
 import {batchActions} from 'redux-batched-actions';
 import {Client4} from 'client';
 import {UserTypes} from 'action_types';
 import {logError} from './errors';
+
+import type {Client4Error} from '../types/errors';
+import type {GenericAction, ActionResult, DispatchFunc, GetStateFunc} from '../types/actions';
```

Comment:
Can you switch these to using non-relative paths, e.g. `from 'types/actions';`? Makes it easier to refactor/move files around